assets/javascripts/modules/autoComplete.js

```diff
@@ -379,15 +381,11 @@ var getSuggestions = function (el) {
 
 /**
   * Find input to post value back to
+  * @param el
 */
 var getTargetInput = function (el) {
-  var target;
-  // input to post back a selection from the autocomplete
-  var dataPostTo = $('[id="' + el.attr("data-target-elem") + '"]');
-  if(dataPostTo.length > 0){
-    target = dataPostTo;
-  }
-  return target;
+  var $targets = el.first().data('target-elem') ? el.first().data('target-elem') : $targetInputElem;
```

Comment:
so you need to be mindful of being backwardly compatible. We can't just change the input on a data attribute without updating all the services that send in said input to the data attribute in the old way.